doc/source/train/examples.rst

```diff
@@ -7,47 +7,115 @@ Ray Train Examples
    .py files in ray/python/ray/train/examples.
 
 Below are examples for using Ray Train with a variety of models, frameworks, 
-and use cases.
+and use cases. You can filter these examples by the following categories:
```

Comment:
When filtering, is it possible to make the headings no longer show up if there are no examples in that category?

![Uploading image.png…]()

Reply:
It's doable in principle, but requires a bit of JS surgery. I'm taking notes and do this as follow-up.

p.s.: arguably, with a proper tag system you don't need the headings in the first place. many other tools just go for a paginated list of examples, see e.g. https://pytorch.org/tutorials/